docs/usage_overview.rst

```diff
@@ -184,7 +184,9 @@ These are as follows:
     define how the sub-folders are separated.  Common values include ".", "/" or "|", but
     this varies by the mail server.  Unfortunately, this isn't a value we can determine
     automatically.  Either check the documentation for your mail server, or check for
-    errors in the logs and try different folder separator values.
+    errors in the logs and try different folder separator values.  If possible,
```

Comment:
"errors in the logs" being paperless-ngx webserver logs? With this PR what do you think about changing these lines to something like:

"Check the documentation for your mail server. In the event of an error when fetching a certain folder, check the Paperless logs for a listing of valid folders and separator values from your account."